internal/pkg/aws/ecs/ecs.go

```diff
@@ -319,6 +319,29 @@ func (e *ECS) HasDefaultCluster() (bool, error) {
 	return true, nil
 }
 
+// ActiveClusters returns the subset of arns that have an ACTIVE status.
+func (e *ECS) ActiveClusters(arns ...string) ([]string, error) {
+	resp, err := e.client.DescribeClusters(&ecs.DescribeClustersInput{
+		Clusters: aws.StringSlice(arns),
+	})
+	switch {
+	case err != nil:
+		return nil, fmt.Errorf("describe clusters: %w", err)
+	case len(resp.Failures) > 0:
+		return nil, fmt.Errorf("describe clusters: %s", resp.Failures[0].GoString())
```

Comment:
Why do we need this 🤔

Reply:
just in case the describe fails for any of the ARNs - seems reasonable to error out in that case? do you think we should just ignore it? 

the only case i can realistically see something like this occurring is in this scenario:
1. we get the service ARNs by tag
2. ECS _really_ deletes the service
3. we call describe services here with the ARN that is now deleted

Reply:
Ohh I guess I didn't realize in this case `DescribeClusters` would actually return `nil` err and put the err in the response.